scripts/submit-analytics.js

```diff
@@ -0,0 +1,201 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+const commandLineArgs = require("command-line-args");
+const fs = require("fs");
+
+let doLog; // placeholder for logger function
+
+class Webhook {
+  /**
+   * Path and credentials required to submit analytics through the webhook (production mode).
```

Comment:
Do we no longer have the option to submit to a staging bucket for testing?

Reply:
I don't think so but I can check.